java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -121,4 +123,21 @@ public void lastsave() {
         var response = client.exec(new Transaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void zrank_zrevrank_withscores() {
+        String zSetKey1 = "{key}:zsetKey1-" + UUID.randomUUID();
+        Transaction transaction = new Transaction();
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0")) {
```

Comment:
```suggestion
        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0"));
```
And move it to the top